rpc/api.proto

```diff
@@ -743,3 +743,53 @@ message VerifyMessageRequest {
 message VerifyMessageResponse {
 	bool valid = 1;
 }
+
+service DecodeMessageService {
+	rpc DecodeRawTransaction (DecodeRawTransactionRequest) returns (DecodeRawTransactionResponse);
+}
+
+message DecodedTransaction {
+	message ScriptSig {
+		string asm = 1;
+		string hex = 2;
+	}
+	message ScriptPubKeyResult {
+		string asm = 1;
+		string hex = 2;
+		int32 req_sigs = 3;
+		string type = 4;
+		repeated string addresses = 5;
+		int64 commit_amt = 6;
+	}
+	message Vin {
+		string coinbase = 1;
```

Comment:
i have no idea what this field, as well as the stakebase field below, are for, given that they're string types.